sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporter.java

```diff
@@ -30,8 +29,8 @@ private AzureMonitor() {
      * @param autoConfigurationCustomizer The OpenTelemetry autoconfiguration to set up.
      */
     public static void customize(AutoConfigurationCustomizer autoConfigurationCustomizer) {
```

Comment:
is it better to put AzureMonitorExporterOption as a second parameter? 
Customer can have their own configuration they want.